### PR TITLE
Support GM Tier0 GW interface

### PR DESCRIPTION
**Pull request by asarfaty**

### CODE REVIEW

**Comment by asarfaty** [comment]
@annakhm  - not sure what you mean about the location. Isn't is already a part of the GW edge cluster / localeServices?

File: nsxt/resource_nsxt_policy_tier0_gateway_interface.go [hunk 1]
```diff
@@ -119,20 +121,47 @@ func resourceNsxtPolicyTier0GatewayInterfaceCreate(d *schema.ResourceData, m int
...
+		localeServices, err := listPolicyTier0GatewayLocaleServices(connector, tier0ID, true)
+		if err != nil {
+			return err
+		}
+		if len(localeServices) == 0 {
+			return fmt.Errorf("Edge cluster is mandatory on gateway %s in order to create interfaces", tier0ID)
+		}
+		for _, objInList := range localeServices {
+			if objInList.EdgeClusterPath != nil {
+				localeServiceID = *objInList.Id
```
Comment (annakhm) [inline nsxt/resource_nsxt_policy_tier0_gateway_interface.go:135]: Here the first locale service that has edge cluster is taken for the interface. But I think we should allow the user to specify the site.

Comment (asarfaty) [reply nsxt/resource_nsxt_policy_tier0_gateway_interface.go]: Done

**Comment by asarfaty** [comment]
Can we merge this one? or are we still missing something here?

**Comment by asarfaty** [comment]
I'll merge it now and update the tests later. Thanks

**Pull request merged**